src/main/java/de/zalando/tip/zalenium/proxy/SauceLabsRemoteProxy.java

```diff
@@ -31,21 +32,18 @@
 
 public class SauceLabsRemoteProxy extends DefaultRemoteProxy {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2160)